docs/readmes/howtos/troubleshooting/datapath_connectivity.md

```diff
@@ -196,6 +196,29 @@ on UE or the server (on SGi side of the network) while debugging the issue.
     the trace output in a github issue.
 15. If this document does not help to debug the issue, please post output of
```

Comment:
Why shouldn't your new steps go above this one, i.e. "If this document does not help to debug the issue..."? Shouldn't this be the last item in the steps in case nothing else works?